tests/active/active_loop_test.py

```diff
@@ -99,5 +104,27 @@ def test_sad(max_sample, expected):
     assert len(dataset) == 10 + expected
 
 
+def test_file_saving(tmpdir):
+    tmpdir = str(tmpdir)
+    heur = heuristics.BALD()
+    ds = MyDataset()
+    dataset = ActiveLearningDataset(ds, make_unlabelled=lambda x: -1)
+    active_loop = ActiveLearningLoop(dataset,
+                                     get_probs_iter,
+                                     heur,
+                                     uncertainty_folder=tmpdir,
+                                     ndata_to_label=10,
+                                     dummy_param=1)
+    dataset.label_randomly(10)
+    _ = active_loop.step()
+    assert len(os.listdir(tmpdir)) == 1
+    file = pjoin(tmpdir, os.listdir(tmpdir)[0])
+    assert "pool=90" in file and "labelled=10"
```

Comment:
the part after and is always true, no?

Reply:
It depends on how many items are labelled